Entity Explorer - IP Address.ipynb

```diff
@@ -6,1891 +6,573 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    ip_address = "147.78.47.75"
</pre><p>I think we should make the default an internal IP</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>